pvr.nextpvr/changelog.txt

```diff
@@ -1,3 +1,13 @@
+
+v22.2.0
+- Port PR 266 from Omega
+- Start timeshift in realtime for radio playback (266)
+- Add support for episode and episode part parsing (266)
+- Clean up duplicated S/E information sent from NextPVR in the subtitle when no subtitle is present. (266)
```

Comment:
Aren't you explaining in line 4 to 6 what PR2666 was about? Then you could remove line 3, which says nothing to normal end users.